README.md

```diff
@@ -1,2 +1,54 @@
 # sumo-project
+
 This repository is used to track anything related to the platform that powers SUMO
+
+## Engineering Board
+
+Short description of the workflow and of the columns in the board.
+
+### Labels
+
+Each label in this board signifies a specific project.
+
+### Columns
+
+#### Backlog column
+
+A bakclog of tasks that will be worked in the next three months.
+Ideally meta-issues should be used here.
+
+#### Next items column
+
+Smaller prioritized backlog with cards that will be worked on in the two upcoming weeks. Should be sparsely populated and frequently updated.
+
```

Comment:
Do we need a triage/icebox column for things which aren't necessarily a priority, but come up while we're working, and might make a good contributor issue (for instance)?